### PR TITLE
JobDescription  CRUD API pushed,Eligibility Criteria and Employment Type API done

### PR DESCRIPTION
**Pull request by balani98**

### CODE REVIEW

File: ARMS_DB_Scripts/images.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    UPDATE ARMS.Images
+    SET modifiedAt  = sysdatetime()
+    WHERE id IN (SELECT DISTINCT id FROM Inserted)
+
+--insert command
+GO
+INSERT into ARMS.Images(employeeId,createdBy,modifiedBy,imageBlobData)
+SELECT '1','deepsy','deepsy', BulkColumn
+ FROM OPENROWSET(Bulk 'C:\Users\deepanshu.balani\Pictures\login.jpg', SINGLE_BLOB) AS BLOB
+ --select command
```
Comment (sonalichawla16) [inline ARMS_DB_Scripts/images.sql:30]: What is this ?

Comment (balani98) [reply ARMS_DB_Scripts/images.sql:30]: This is command to insert blobdata

File: arms-services/src/Arms.Api/appsettings.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "Db": "server=CYG263\\SQLEXPRESS;database=ARMS_DB;Trusted_Connection=True;CurrentSchema=ARMS;"
+
+    "Db": "server=CYG363; database=ARMS_DB; Trusted_Connection=True;"
 
   },
```
Comment (sonalichawla16) [inline arms-services/src/Arms.Api/appsettings.json:6]: Never commit this

File: arms-services/src/Arms.Api/Controllers/JobDescriptionController.cs [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                var resNull = new
+                {
+                    success = "true",
+                    payload = new
+                    {
+
+                        message = "This Jobdescription does not exist with this ID"
+                    }
+
+                };
```
Comment (sonalichawla16) [inline arms-services/src/Arms.Api/Controllers/JobDescriptionController.cs:159]: This API doesnt make any sense to me

File: arms-services/src/Arms.Api/appsettings.json (hunk 2 above)
Comment (balani98) [reply arms-services/src/Arms.Api/appsettings.json]: connection string file is removed from here

File: arms-services/src/Arms.Api/Controllers/JobDescriptionController.cs (hunk 3 above)
Comment (balani98) [reply arms-services/src/Arms.Api/Controllers/JobDescriptionController.cs:266]: function name changed

**Comment by balani98** [comment]
All conflicts resolved,you can merge

**Pull request merged**